src/resources/users/resources/catalog-requests/entities/contact-info.entity.ts

```diff
@@ -0,0 +1,40 @@
+import { ApiExtraModels, ApiProperty } from '@nestjs/swagger';
+import { InferModel } from 'drizzle-orm';
+import catalogRequestContactInfo from '../../../../../drizzle/schema/catalog_request_contact_info';
+import { OptionalApiProperty } from '../../../../../openapi/decorators';
+
+export class GeoLocation {
```

Comment:
If you define a class multiple times, then it would behave as a different entity, extrapolate the class into a separate file and import it in multiple places